website/content/en/preview/AWS/provisioning.md

```diff
@@ -6,6 +6,36 @@ description: >
   Learn AWS-specific Karpenter provisioning settings
 ---
 
+Provisioner settings specific to Karpenter for the AWS cloud provider are described here.
+The following example shows optional and required settings for a Karpenter provisioner for AWS:
+
+
+```yaml
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  requirements:                      
+    - key: karpenter.sh/capacity-type         # optional, set to on-demand by default
+      operator: In
+      values: ["spot"]

```

Comment:
I'm not sure if this is the right place to put this but a question that might come up based on the default being on-demand is: "what happens when both are in the list of values" which in that case, we'd default to spot.